pytest_fixtures/core/sat_cap_factory.py

```diff
@@ -367,8 +366,6 @@ def installer_satellite(request):
         snap=settings.server.version.snap,
     )
     sat.install_satellite_or_capsule_package()
-    installed_version = sat.execute('rpm --query satellite').stdout
-    assert sat_version in installed_version
```

Comment:
Would you please move the assertion into a test as you have indicated in PR title instead of removing the assertion completely ?

Reply:
@jyejare To `tests.foreman.installer.test_installer.test_satellite_installation` specifically.

Reply:
Great!